swarm/network/stream/delivery_test.go

```diff
@@ -55,9 +59,20 @@ func TestStreamerRetrieveRequest(t *testing.T) {
 	)
 	streamer.delivery.RequestFromPeers(ctx, req)
 
+	stream := NewStream(swarmChunkServerStreamName, "", true)
+
 	err = tester.TestExchanges(p2ptest.Exchange{
 		Label: "RetrieveRequestMsg",
 		Expects: []p2ptest.Expect{
+			{
```

Comment:
The test name is the same, so why do the expectations change? (And why are the message codes literals?)

Reply:
yes an explanation would be nice: the reason is that before doServeRetrieve and doRetrieve were independently set so when testing a server you could just set doRetrieve to false which resulted in the server node not sending a subscription. Since this is not an existing/allowed scenario now, the test needs to start with expecing a subscription

Reply:
and yes, message code literals should be used instead of integers. 
but everywhere. so let that be another PR

Reply:
Added some short descriptions, nevertheless the whole test file was a bit short on comments. I added a few, but it could be subject of a separate PR to improve this is more is needed.